WalletWasabi.Gui/Tabs/WalletManager/GenerateWalletViewModel.cs

```diff
@@ -26,10 +26,7 @@ public GenerateWalletViewModel(WalletManagerViewModel owner) : base("Generate Wa
 		{
 			Owner = owner;
 
-			GenerateCommand = ReactiveCommand.Create(() =>
-			{
-				DoGenerateCommand();
-			},
+			GenerateCommand = ReactiveCommand.Create(DoGenerateCommand,
```

Comment:
semicolon and parenthesis needed

Reply:
Create method takes two arguments, this method is the first, second is on the next line which probably made the diff look weird.

Reply:
Oh I see now, my bad

Reply:
No offense, but better to let more technically qualified people review the code, otherwise bugs could be introduced.